### ISSUE TITLE
Security sunglasses spawn on the terminal. Somehow.

### ISSUE THREAD

**Issue by TsjipTsjip** [Sector-Umbra/Sector-Umbra#129]
## Description

Whenever I late join there's like 3 security sunglasses on the seats in the arrivals terminal. I got no clue why they're there.

**Reproduction**

Hard. Has to be observed and tracked down in an actual round first to narrow down reproduction steps.

**Additional context**

This might be a security loadout issue.

**Comment by luckyshotpictures** [comment]
I believe they spawn either at a cryopod on station, or on arrivals depending on what the player has chosen.

**Comment by onenineone** [comment]
Yes, I believe it's because of the eyewear section for security members in the loadouts. Security members naturally spawn in with security glasses and the second pair that cannot be turned off or altered to be the security hud spawns on the ground because there are no free slots on the eyes. That's my guess.

**Comment by Kukutis96513** [comment]
> Yes, I believe it's because of the eyewear section for security members in the loadouts. Security members naturally spawn in with security glasses and the second pair that cannot be turned off or altered to be the security hud spawns on the ground because there are no free slots on the eyes. That's my guess.

I think you can set it to the security hud, it just doesnt change the loadout sprite for some reason.

Anyways does anyone know how to reproduce this bug?

**Comment by PursuitInAshes** [comment]
Is this issue still present? I haven't seen it, but I haven't been around ingame recently. If its still present, anyone have steps to recreate?

**Comment by TsjipTsjip** [comment]
It still is present yeah. People just pick 'em up and take them to security right now.

**Comment by dffdff2423** [comment]
Strangely I have never encountered this on CD (nor has anybody complained about it). Are the reproduction steps just spawn as a secoff with the glasses in your loadout?
